.github/workflows/weekly.yml

```
@@ -1,8 +1,8 @@
-name: fortnightly tests
+name: weekly tests
 
 on:
   schedule:
-  - cron: 17 7 1,15 * *
+  - cron: 17 7 * * 2
```

I like how this would run in the 48h before our telecon. I wonder about the timezone here, though. Probably UTC.

Could do this to be on the safe side:
```suggestion
  - cron: 17 7 * * 1
```